### DISCUSSION TITLE
what's the difference between I2SStream out; AudioKitStream out; ?

### DISCUSSION THREAD

**Discussion by carbonadam** [pschatzmann/arduino-audio-tools#474] [Q&A] [answered]
Hi Phil,  Just getting into understanding you libraries and I very much appreciate the work you have done.  I've been trying to use my Lyrat board ´with i2s and while i can get it to work with AudioKitStream out;  the default setting for I2SStream out doesn't seem to playback anything.  I've got one of the Maximillian examples running now using 
AudioKitStream out; but it sounds a little crunchy. I had been using an external board earlier this year (same as your examples the pcm5102) and I can vaguely remember  got some clean audio out.  Is there any difference between the 2 0utput methods?  and whats your thoughts on the 8288 with sound quality?

All the best
Adam

**Comment by pschatzmann** [comment, marked as answer]
AudioKitStream is basically I2SStream with some additional logic to set up the codec.
I didn't notice any issues: Did you set the logger to Warning ?

**Reply by carbonadam** [reply]
ok thanks for the clarification.  I didn't change anything from the 05-fm1 except I tried to disable the serial output to see if that helps but no luck so far.  Not sure how to set the logger to warning to see any errors but I can look into that.

Thanks for such a fast reply too:)

**Comment by carbonadam** [comment]
not having any luck to get output via I2SStream out;
Where are the settings to make sure its using the lyrat or right pins?  I keep trying some of those Maximillian examples but they sure dont sound like the names of the file examples.
I have the esp32 rover module selected in the boards manager

**Comment by pschatzmann** [comment]
Of cause I2SStream will not work! These boards only work when the audio chip is configured in accordance with the i2s settings...
That's what AudioKitStream is for.
